### DISCUSSION TITLE
HUSBZB-1 has wrong product description after firmware update

### DISCUSSION THREAD

**Discussion by ProjectPatatoe** [zwave-js/zwave-js#3045] [Q&A] [answered]
I didn't open an issue yet because I don't know what the mechanism is for getting the product description. Or more precisely, if this is zwave-js's fault or the firmware upgrade's fault.

I did a firmware update of the HUSBZB-1 usb zwave/zigbee stick firmware and zwave-js identifies it as 

Manufacturer | Product | Product code|Z-Wave Plus
|----|----|----|----|
Sigma Designs (Former Zensys) | Smart Temperature & Humidity Sensor | M417_9E|

Unfortunately, I don't remember what it said before but it is obviously not a temp/humidity sensor. Where is it getting the product description and product code from? Is this pulled directly from the device or is it some table?

The 'PC controller' from SiLabs I don't think really told me those so it feels like a table in zwave-js.

**Comment by AlCalzone** [comment, marked as answer]
Looks like the update changed the product fingerprint from [0x0000:0x0001:0x0001](https://devices.zwave-js.io/?jumpTo=0x0000:0x0001:0x0001:0.0) to [0x0000:0x0003:0x0008](https://devices.zwave-js.io/?jumpTo=0x0000:0x0003:0x0008:0.0).

We should probably merge these two files into one, although I'm sure that someone will complain then that his `M417_9E` is identified as an `HUSBZB-1` 🤷🏻‍♂️

**Reply by AlCalzone** [reply]
Let's continue this in https://github.com/zwave-js/node-zwave-js/issues/3047
